### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by Feliverse**
### Learning objectives

- Use callbacks and promises.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.
- Use webpack to bundle JavaScript.

### Project requirements

- IMPORTANT NOTE This is the first time I will use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Create an npm project with webpack. 
- Write plain HTML markup with minimum styling (just to make the layout)
image

- 
Don't make API calls.
- Use ES6 modules, with import and export.

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @Feliverse, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - Unfortunately, your project does not have a proper README file following the Microverse README template. Kindly [use this template](https://github.com/microverseinc/readme-template) and make sure you delete the Microverse logo and the live demo section if the project does not have one, and make sure you customize all the sections to match this specific project details. Also, your PR description should not be a copy/paste of the project requirements. The project requirements were written for a user to follow and have some details that are not important for a reviewer. Kindly write a proper PR summary describing the introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+<a name="readme-top"></a>
```
Comment (addod19) [inline src/README.md]: - [x] Nice job adding the README file, however, it's in the wrong position, kindly move it to the root directory and remove the previous README file from there

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @Feliverse, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [x] Correct Git-flow followed :+1: 
- [x] Node_modules added to the gitignore file :100: 
- [x] All webpack related packages added :fireworks: 
- [x] Basic project structure :1st_place_medal: 
- [x] Code matches up with the wireframe :+1: 
- [x] The command `npm start` runs the app as expected :rocket: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Feliverse 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 Correct Git-flow followed 👍
 Node_modules added to the gitignore file 💯
 All webpack related packages added 🎆
 Basic project structure 🥇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/README.md (hunk 1 above)
Comment (Feliverse) [reply src/README.md]: Thank you, I will move it just now

**Pull request merged**